### DISCUSSION TITLE
figma.com

### DISCUSSION THREAD

**Discussion by PopCat19** [catppuccin/catppuccin#2459] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

Stylus/userstyles.css theming the website's UI in catppuccin flavors.

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by Readf0x** [comment]
It might be possible to leech off of [Figma-Linux/figma-linux](https://github.com/Figma-Linux/figma-linux)

**Comment by WalkQuackBack** [comment]
Has anyone picked this up?

**Comment by WalkQuackBack** [comment]
I'm going to be picking this up as a userstyle.

**Comment by WalkQuackBack** [comment]
Made a PR: https://github.com/catppuccin/userstyles/pull/2047

**Comment by sgoudham** [comment]
Closing as a PR has been raised for the userstyle
